src/package_io/parse_input.star

```diff
@@ -66,78 +83,292 @@ def parse_input(plan, input_args):
-                    "tx_spammer_extra_args"
-                ],
-            ),
-            launch_additional_services=result["launch_additional_services"],
-            additional_services=result["additional_services"],
-            wait_for_finalization=result["wait_for_finalization"],
-            global_client_log_level=result["global_client_log_level"],
-            mev_type=result["mev_type"],
+            num_validator_keys_per_node=result["network_params"][
+                "num_validator_keys_per_node"
+            ],
+            network_id=result["network_params"]["network_id"],
+            deposit_contract_address=result["network_params"][
+                "deposit_contract_address"
+            ],
+            seconds_per_slot=result["network_params"]["seconds_per_slot"],
+            slots_per_epoch=result["network_params"]["slots_per_epoch"],
+            genesis_delay=result["network_params"]["genesis_delay"],
+            capella_fork_epoch=result["network_params"]["capella_fork_epoch"],
+            deneb_fork_epoch=result["network_params"]["deneb_fork_epoch"],
+            electra_fork_epoch=result["network_params"]["electra_fork_epoch"],
         ),
-        result,
+        mev_params=struct(
+            mev_relay_image=result["mev_params"]["mev_relay_image"],
+            mev_builder_image=result["mev_params"]["mev_builder_image"],
+            mev_boost_image=result["mev_params"]["mev_boost_image"],
+            mev_relay_api_extra_args=result["mev_params"]["mev_relay_api_extra_args"],
+            mev_relay_housekeeper_extra_args=result["mev_params"][
+                "mev_relay_housekeeper_extra_args"
+            ],
+            mev_relay_website_extra_args=result["mev_params"][
+                "mev_relay_website_extra_args"
+            ],
+            mev_builder_extra_args=result["mev_params"]["mev_builder_extra_args"],
+            mev_flood_image=result["mev_params"]["mev_flood_image"],
+            mev_flood_extra_args=result["mev_params"]["mev_flood_extra_args"],
+            mev_flood_seconds_per_bundle=result["mev_params"][
+                "mev_flood_seconds_per_bundle"
+            ],
+            launch_custom_flood=result["mev_params"]["launch_custom_flood"],
+        ),
+        tx_spammer_params=struct(
+            tx_spammer_extra_args=result["tx_spammer_params"]["tx_spammer_extra_args"],
+        ),
+        launch_additional_services=result["launch_additional_services"],
+        additional_services=result["additional_services"],
+        wait_for_finalization=result["wait_for_finalization"],
+        global_client_log_level=result["global_client_log_level"],
+        mev_type=result["mev_type"],
+        snooper_enabled=result["snooper_enabled"],
+        parallel_keystore_generation=result["parallel_keystore_generation"],
     )
 
 
+def parse_network_params(input_args):
+    result = default_input_args()
+    for attr in input_args:
```

Comment:
I think that you could merge this for loop with this one: https://github.com/kurtosis-tech/eth2-package/pull/228/files#diff-482051a627cba885ddfdf14a163ec475933d35428175cb1060a1eb1d2207afc3R65

Reply:
I didn't want to change code too wildly for merge; but I will track it as a TODO. Not adding a TODO immediately as a comment as I want to remain on the last commit as long as possible as people are running tests.

Reply:
https://github.com/kurtosis-tech/eth2-package/issues/229 for now